CMakeLists.txt

```diff
@@ -96,6 +96,7 @@ SET(KokkosKernels_INSTALL_TESTING OFF CACHE INTERNAL
 IF (KokkosKernels_INSTALL_TESTING)
   # Force testing on if we are doing intall testing
   SET(KOKKOSKERNELS_ENABLE_TESTS ON)
+  SET(KOKKOSKERNELS_ENABLE_BENCHMARKS ON)
```

Comment:
i dont think we want to run the perf test when doing install testing

Reply:
:+1:  d953f8f75e029f5929d4c02c3c3c8319a413cea3